views/entry/module-displaydate.tt

```diff
@@ -74,12 +74,22 @@ the same terms as Perl itself.  For a copy of the license, please reference
 
     <div class="row">
         <div class="columns">
-            [%- form.checkbox_nested( label = dw.ml( ".label.autoupdate" )
-                name = "update_displaydate"
-                id = "js-entrytime-autoupdate"
-
-                value = "1"
-            ) -%]
+            [% IF "$displaydate_check" %]
```

Comment:
So this is completely functional (and sorry for not being available on IRC when you were problem-solving this!)

I'd like to suggest a cleaner alternative:

    default = displaydate_check